onnxtech.txt

```diff
@@ -3,165 +3,394 @@ apis: c
 apis: c#
 apis: c++
 apis: c++: hip
-apis: c++: hipify
 apis: csharp
 apis: java
-apis: java: react
-apis: objective c
+apis: javascript
+apis: javascript: react
+apis: objective-c
 apis: python
 architectures
-architectures: amd
-architectures: apple silicon
-architectures: arm
-architectures: arm: aarch64
-architectures: arm: neon
-architectures: arm32
-architectures: arm64
-architectures: avx
-architectures: avx512
 architectures: cpu
+architectures: cpu: arm
+architectures: cpu: arm: apple silicon
+architectures: cpu: arm: aarch64
+architectures: cpu: arm: neon
+architectures: cpu: x86
+architectures: cpu: x86: extensions
+architectures: cpu: x86: x64
+architectures: cpu: power
+architectures: accelerator
+architectures: accelerator: vpu
+architectures: accelerator: npu
 architectures: gpu
 architectures: gpu: nvidia
-architectures: ibm power
-architectures: ibm power: power10
-architectures: npu
-architectures: vad
-architectures: vpu
-architectures: x64
-architectures: x86
-architectures: x86: amx
-azure
+architectures: gpu: rocm
 execution providers
+execution providers: acl
 execution providers: armnn
+execution providers: azure
+execution providers: cann
 execution providers: coreml
+execution providers: cpu
 execution providers: cuda
-execution providers: cuda: nccl
-execution providers: direct ml
-execution providers: ep
+execution providers: directml
+execution providers: dnnl
+execution providers: js
 execution providers: migraphx
 execution providers: nnapi
-execution providers: onednn
-execution providers: openvino: ov
-execution providers: provider
+execution providers: openvino
+execution providers: qnn
+execution providers: rknpu
 execution providers: rocm
-execution providers: rocm: rocblas
+execution providers: shared
+execution providers: snpe
 execution providers: tensorrt
 execution providers: tvm
-featurizers
+execution providers: vitisai
+execution providers: webnn
+execution providers: winml
+execution providers: xnnpack
 graph
-graph: graph processing: 3d unet
-graph: graph processing: abseil
-graph: graph processing: edge
-graph: graph processing: gist
-graph: graph processing: weight
+graph: graph processing
 graph: optimization
 graph: partition
-graph: protobuf
+graph: serialization
 inferencing
 models
-models: customer model
```

Comment:
I couldn't really find references to customer models, but 'custom models' seemed to cover this and other custom models